### PR TITLE
Ledger types

### PR DESCRIPTION
**Pull request by HenryNguyen5**
Adds types for packages under https://github.com/LedgerHQ/ledgerjs
- `@ledgerhq/hw-app-eth` https://github.com/LedgerHQ/ledgerjs/tree/master/packages/hw-app-eth
- `@ledgerhq/hw-transport-node-hid` https://github.com/LedgerHQ/ledgerjs/tree/master/packages/hw-transport-node-hid
- `@ledgerhq/hw-transport` https://github.com/LedgerHQ/ledgerjs/tree/master/packages/hw-transport
-  `@ledgerhq/hw-transport-u2f` https://github.com/LedgerHQ/ledgerjs/tree/master/packages/hw-transport-u2f

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -129,6 +132,7 @@
     "tslint-react": "3.5.1",
     "types-rlp": "0.0.1",
     "typescript": "2.8.1",
+    "u2f-api": "1.0.6",
```
Comment (wbobeirne) [inline package.json]: Do we need to add this dependency just for types? Or is there a way around that?

Comment (HenryNguyen5) [reply package.json]: Yeah, the package is written in TS so you have to have it as a dep to get the types.

Comment (HenryNguyen5) [reply package.json]: Ideally we can merge these types upstream so it won't have the extra package

**Comment by dternyak** [comment]
In the long-term, it will be nice to have these upstreamed. In the interim, I'll merge.

**Pull request merged**